.github/workflows/build-website.yml

```
@@ -51,16 +51,15 @@ jobs:
           ./ENV/bin/python ./tools/check-links http://localhost:9001/
           kill %1
         shell: bash
-      - name: check links (build)
+      - name: check links (apache2)
         run: |
-          cd website/www/
-          python3.7 -m http.server 9002 &
-          sleep 0.5  # HACK(strager): Wait for server to start.
-          cd ../../
-          ./ENV/bin/python ./tools/check-links http://localhost:9002/
-          kill %1
+          sudo apt-get install apache2 -y
+          sudo mkdir -p /var/www/quick-lint-js.com/www
+          sudo cp -R website/www/* /var/www/quick-lint-js.com/www
+          sudo sed -i "s,/var/www/html,/var/www/quick-lint-js.com/www,g" /etc/apache2/sites-enabled/000-default.conf
```

Why not copy the files into `/var/www/html` so we don't need to `mkdir` or `sed`? If it's because some files already exist in `/var/www/html`, you can use `rsync`'s `--delete` flag:
```suggestion
          sudo rsync --archive --delete website/www/ /var/www/html/
```